app/models/mixins/process_tasks_mixin.rb

```diff
@@ -68,6 +68,8 @@ def invoke_tasks_remote(options)
         begin
           remote_connection = InterRegionApiMethodRelay.api_client_connection_for_region(region, remote_options[:userid])
           invoke_api_tasks(remote_connection, remote_options)
+        rescue ManageIQ::API::Client::ResourceNotFound => err
```

Comment:
I think this is a good way to rescue the error, but wouldn't this bail out before all of the resources were processed in `invoke_api_tasks`?

Why not leave this rescue inside the resource loop in that method?